Sources/FuzzilliCli/main.swift

```diff
@@ -26,72 +26,74 @@ Usage:
+                                      Only the mutation engine should be regarded stable at this point.
+    --corpus=name                   : The corpus scheduler to use. Available schedulers: "basic" (default), "markov"
+    --minDeterminismExecs=n         : The minimum number of times a new sample will be executed when checking determinism (default: 3)
+    --maxDeterminismExecs=n         : The maximum number of times a new sample will be executed when checking determinism (default: 50)
+    --noDeterministicCorpus         : Don't ensure that samples added to the corpus behave deterministically.
+    --maxResetCount=n               : The number of times a non-deterministic edge is reset before it is ignored in subsequent executions.
+                                      Only used as part of --deterministicCorpus.
+    --logLevel=level                : The log level to use. Valid values: "verbose", info", "warning", "error", "fatal"
+                                      (default: "info").
+    --numIterations=n               : Run for the specified number of iterations (default: unlimited).
+    --timeout=n                     : Timeout in ms after which to interrupt execution of programs (default: 250).
+    --minMutationsPerSample=n       : Discard samples from the corpus only after they have been mutated at least this many times (default: 100).
+    --minCorpusSize=n               : Keep at least this many samples in the corpus regardless of the number of times
+                                      they have been mutated (default: 1024).
+    --maxCorpusSize=n               : Only allow the corpus to grow to this many samples. Otherwise the oldest samples
+                                      will be discarded (default: unlimited).
+    --markovDropoutRate=p           : Rate at which low edge samples are not selected, in the Markov Corpus Scheduler,
+                                      per round of sample selection. Used to ensure diversity between fuzzer instances
+                                      (default: 0.10)
+    --consecutiveMutations=n        : Perform this many consecutive mutations on each sample (default: 5).
+    --minimizationLimit=p           : When minimizing interesting programs, keep at least this percentage of the original instructions
+                                      regardless of whether they are needed to trigger the interesting behaviour or not.
+                                      See Minimizer.swift for an overview of this feature (default: 0.0).
+    --storagePath=path              : Path at which to store output files (crashes, corpus, etc.) to.
+    --resume                        : If storage path exists, import the programs from the corpus/ subdirectory
+    --overwrite                     : If storage path exists, delete all data in it and start a fresh fuzzing session
+    --exportStatistics              : If enabled, fuzzing statistics will be collected and saved to disk every 10 minutes.
```

Comment:
Maybe replace the `every 10 minutes` part with something like ` in regular intervals`